ex02.ex

```diff
@@ -41,7 +42,8 @@ defmodule Ex02 do
   # Do the same using the & syntax #
   ##################################
 
-  list2b = your_anonymous_function(1, 2)
+  #list2b = your_anonymous_function(1, 2)
+  list2b = &([&1 + &2, &1 - &2])
```

Comment:
No deduction, but I find this code easier to read without the unnecessary parentheses:

~~~ elixir
list2b = &[ &1 + &2, &1 - &2 ]
~~~